wallet/src/main/java/com/android/identity_credential/wallet/navigation/WalletNavigation.kt

```diff
@@ -75,7 +76,8 @@ fun WalletNavigation(
                 qrEngagementViewModel = qrEngagementViewModel,
                 cardViewModel = cardViewModel,
                 settingsModel = application.settingsModel,
-                permissionTracker = permissionTracker
+                permissionTracker = permissionTracker,
+                capabilities = AndroidKeystoreSecureArea.Capabilities(context = application.applicationContext),
```

Comment:
I don't think it's wise to calculate the capabilities and pass them. First of all, you shouldn't be doing anything expensive in your composable and this is indeed rather expensive to calculate. Second, the value of might change as users add the Screenlock later but we are not guaranteed that we'll create a new `Capabilities` object with these capabilities.

Since we only need these when the user presses the "Add new credential" button (and we have two of them) we can defer the construction to then and just do it in the `onClick` handler and that way you also won't have problems with stale data (I think this is what I originally proposed).

Reply:
ok - the reason I had the `Capabilities` object created in the navigation (which, you're correct, could go stale since it's only made when navigation to the main screen) was to avoid passing context into the composable